### PR TITLE
adding microsite awareness email backend to identify distinct sender …

### PR DESCRIPTION
**Pull request by diegomillan**
…domains. Now emails are delivered through mailgun.

These are requiered configuration tokens
env tokens
EMAIL_BACKEND = 'edunext.smtpbackend.MicrositeEmailBackend'
EMAIL_HOST = 'smtp.mailgun.org'
EMAIL_PORT = 587

### CODE REVIEW

File: common/djangoapps/edunext/smtpbackend.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Microsite configuration email backend module.
+
+Contains the class for microsite email backend.
+
+"""
+
+from django.core.mail.backends.smtp import EmailBackend
+from openedx.conf import settings
+
+class MicrositeEmailBackend (EmailBackend):
```
Comment (felipemontoya) [inline common/djangoapps/edunext/smtpbackend.py]: Could you please name it MicrositeAware...

File: common/djangoapps/edunext/smtpbackend.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+"""
+Microsite configuration email backend module.
+
+Contains the class for microsite email backend.
+
+"""
+
+from django.core.mail.backends.smtp import EmailBackend
+from openedx.conf import settings
+
```
Comment (felipemontoya) [inline common/djangoapps/edunext/smtpbackend.py]: This is probably not pep8

**Comment by felipemontoya** [comment]
@diegomillan some minor inline comments

**Comment by diegomillan** [comment]
@felipemontoya now 
EMAIL_BACKEND = 'edunext.smtpbackend.MicrositeAwareEmailBackend'

**Pull request merged**